apps/opik-frontend/Dockerfile

```diff
@@ -21,7 +21,9 @@ FROM nginx:alpine
 # Copy the built files from the builder stage
 COPY --from=builder /opt/frontend/dist /usr/share/nginx/html
 
+RUN sed -i '/access_log.*main/d' /etc/nginx/nginx.conf
```

Comment:
@obezpalko  why doing it here and not having a better logic in the nginx template to have the `access_log` or not depending on what we're trying to achieve here? https://github.com/comet-ml/opik/blob/main/deployment/helm_chart/opik/templates/configmap-frontend-nginx.yaml#L63

Reply:
we do not have nginx.conf in templates. It comes from original nginx image with defined access_log.
In our templates we also have access_log definition which causes duplicates in log records with different formats